src/VR.js

```diff
@@ -605,19 +604,30 @@ class FakeVRDisplay extends VRDisplay {
 
 const createVRDisplay = () => new FakeVRDisplay();
 
-const getGamepads = (() => {
+const getGamepads = function(window) {
+
+  let globalWindow = window;
```

Comment:
Is there a stylistic reason for renaming window here?

Reply:
`window` is a pretty overloaded variable name. Intent was to convey that this is something different than your usual global `window` reference.

Reply:
renamed